qlib/contrib/backtest/account.py

```diff
@@ -104,10 +104,9 @@ def update_daily_end(self, today, trader):
             # if suspend, no new price to be updated, profit is 0
             if trader.check_stock_suspended(code, today):
                 continue
-            else:
```

Comment:
Why change this?

Reply:
The results are the same while logic is simpler and reduced one line of code.